RUFAS/end_to_end_tester.py

```diff
@@ -0,0 +1,91 @@
+from collections import namedtuple
+from deepdiff import DeepDiff
+from pathlib import Path
+from typing import Any
+
+from RUFAS.output_manager import OutputManager
+from RUFAS.units import MeasurementUnits
+
+import json
+import re
+
+
+TestResultPaths = namedtuple("TestResultPaths", ["domain_prefix", "expected_results_path", "actual_results_path"])
+
+
+"""
+Maps a RuFaS domain that is supported for end-to-end testing to information needed to collect actual and expected
+results for testing that domain, and the prefix used when saving the test results to the Output Manager.
+"""
+DOMAINS = {
+    "Feed Storage": TestResultPaths(
+        "FeedStorageResults",
+        "input/data/end_to_end_testing/e2e_json_feed_storage_filter.json",
+        "end-to-end-testing_saved_variables_e2e_feed_storage_",
+    )
+}
```

Comment:
More descriptive name plus type annotation 
```suggestion
TEST_DOMAINS: Dict[str, TestResultPathType] = {
    "Feed Storage": TestResultPaths(
        "FeedStorageResults",
        "input/data/end_to_end_testing/e2e_json_feed_storage_filter.json",
        "end-to-end-testing_saved_variables_e2e_feed_storage_",
    )
}
```

Also, the key of this dict seems redundant, `domain_prefix` serves the same purpose. 
```suggestion
TEST_DOMAINS: List[TestResultPathType] = [
    TestResultPaths("FeedStorageResults",
        "input/data/end_to_end_testing/e2e_json_feed_storage_filter.json",
        "end-to-end-testing_saved_variables_e2e_feed_storage_",
    )
]
```
Furthermore, why not getting these from IM instead of hardcoding?

Reply:
> Also, the key of this dict seems redundant, domain_prefix serves the same purpose.

They could pretty easily be substituted for one another. That said, I do like having them separate because one makes for a really log message that reads really nicely and the other makes for a variable output prefix that looks really good.

Edit: scratch that, it makes them easier to incorporate into the inputs if they don't have a key.

Reply:
> Furthermore, why not getting these from IM instead of hardcoding?

I imagine that once we get the model up and running, these domains will be pretty set and we won't need to update them often. In which case it makes more sense to me to have them hardcoded, as opposed to being a modifiable input file. Other than that though, no good reason. I'll update it to use an input file and the input manager